azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/utils/FTPUtils.java

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+package com.microsoft.azure.maven.webapp.utils;
+
+import org.apache.commons.net.ftp.FTP;
+import org.apache.commons.net.ftp.FTPClient;
+import org.apache.maven.plugin.logging.Log;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Stack;
+
+// Todo: Merge this class with FTPUploader in lib to reduce duplicate
+public class FTPUtils {
+
+    public static final String REPLY_MESSAGE = "Reply Message : %s";
+    public static final String FAILED_TO_UPLOAD_RESOURCE = "Failed to upload file: ";
+
+    public static FTPClient getFTPClient(final String ftpServer, final String username, final String password)
+            throws IOException {
+        final FTPClient ftpClient = new FTPClient();
+        ftpClient.connect(ftpServer);
+        ftpClient.login(username, password);
+        ftpClient.setFileType(FTP.BINARY_FILE_TYPE);
+        ftpClient.enterLocalPassiveMode();
+        return ftpClient;
+    }
+
+    public static void uploadFile(final FTPClient ftpClient, final String sourceFilePath,
+                                  final String targetFilePath, Log log) throws IOException {
+        log.info(String.format("Uploading resource %s to %s", sourceFilePath, targetFilePath));
```

Comment:
"Uploading resource %s to %s" to constant

Reply:
Fixed